libretroshare/src/gxs/rsgxsnetservice.cc

```diff
@@ -33,7 +35,7 @@
 //    |
 //    +----------- sharePublishKeys()
 //    |
-//    +----------- syncWithPeers()
+//    +----------- pullFromPeers()
```

Comment:
I dont think this renaming is appropriate: syncWithPeers() sends to all peers the last change time you know for their own data. So you're not pulling anything. If they see that this time is too old (as compared to their own data) they send you the list of new stuff. So it's not pulling either.